bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/converter/type/AbstractTypeConverter.java

```diff
@@ -43,23 +43,23 @@ public abstract class AbstractTypeConverter<T extends State> implements TypeConv
     /**
      * Defines all devices where the state datapoint must be inverted.
      */
-    private static final List<StateInvertInfo> stateInvertDevices = new ArrayList<>(3);
+    private static final List<StateInvertInfo> STATE_INVERT_INFOS = new ArrayList<>(3);
```

Comment:
Maybe this would be clearer to avoid uncountable noun situation?
```suggestion
    private static final List<StateInvertInfo> STATE_INVERT_INFO_LIST = new ArrayList<>(3);
```